frontend/javascripts/components/ListView.jsx

```diff
@@ -15,8 +15,8 @@ class ListView extends React.Component {
   }
 
   componentWillMount() {
-    this.props.listActions.fetchLists(this.props.params.projectID)
-    this.props.listActions.fetchProject(this.props.params.projectID)
+    this.props.listActions.fetchLists(this.props.match.params.projectID)
```

Comment:
__[ESLint]__ <sub>posted by Sider</sub>

Error - 'match' is missing in props validation (react/prop-types)